CHANGELOG.md

```diff
@@ -1,11 +1,52 @@
 # Changelog
 
-**THIS CHANGELOG HAS BEEN DEPRECATED**
+This changelog contains notable changes that make into each [`cardano-node` release](https://github.com/input-output-hk/cardano-node/releases),
+starting with version `8.0`. We also provide per-Haskell-package changelogs. Please see
+the `CHANGELOG.md` for each individual package for any changes relevant for developers.
 
-We have adopted per-Haskell-package changelogs. Please see the `CHANGELOG.md` for each
-individual package for any notable changes.
+If you are looking for the Ledger Releasing and Versioning Process then you can find it in
+[RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd).
 
-See the [REALEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd) for an up-to-date information about Releasing and Versioning Process.
+## 8.2.0
+
+- Restructure certificate interface
+- Conway related changes:
+  - Introduction of a `DRep`, with the default ones that vote No and Abstain.
```

Comment:
Another nitpick:  "Introduce `DRep` with the defaults"  would be more consistent with the list below, but it's really just aesthetics.

Reply:
Perfect. Done